docs/language/accounts.mdx

```diff
@@ -88,6 +88,19 @@ to the `prepare` phase of the transaction.
 
       let keys: AuthAccount.Keys
 
+      // Key management
+
+      // Adds a public key to the account.
+      // The public key must be encoded together with their signature algorithm, hashing algorithm and weight.
+      // This method is currently deprecated and is available only for the backward compatibility.
+      // `keys.add` method can be use instead.
+      fun addPublicKey(_ publicKey: [UInt8])
+
+      // Revokes the key at the given index.
+      // This method is currently deprecated and is available only for the backward compatibility.
+      // `keys.revoke` method can be use instead.
+      fun removePublicKey(_ index: Int)
```

Comment:
I guess these should go away if re-sync with the feature-branch?

Reply:
Ah yea, i will resync everything to fix this.